Makefile

```diff
@@ -696,14 +696,27 @@ nova_deploy_cleanup: ## cleans up the service instance, Does not affect the oper
 
 ##@ KUTTL tests
 
-.PHONY: mariadb_kuttl
-mariadb_kuttl: namespace input openstack_crds deploy_cleanup mariadb_deploy_prep mariadb  ## runs kuttl tests for the mariadb operator. Installs openstack crds and keystone operators and cleans up previous deployments before running the tests.
+.PHONY: mariadb_kuttl_run
+mariadb_kuttl_run: input ## runs kuttl tests for the mariadb operator, assumes that everything needed for running the test was deployed beforehand.
```

Comment:
Is there a specific reason we kept `input` as dependency here?

Reply:
I kept it because I saw that it's the only dependency that is set for all *_deploy targets, so I thought it would be more consistent, but can be removed.

Reply:
I think we can remove it. In the case when the user directly running `mariadb_kuttl_run` and therefore installed all the prerequisites I assume the secret is also created by the user.

Reply:
Makes sense, done.